translations/README.es.md

```diff
@@ -0,0 +1,56 @@
+![Guía freeCodeCamp](https://s3.amazonaws.com/freecodecamp/wide-social-banner.png)
+
+# Guía freeCodeCamp 
+
+[![Build Status](https://img.shields.io/travis/freeCodeCamp/guides/master.svg?style=flat-square)](https://travis-ci.org/freeCodeCamp/guides) [![license](https://img.shields.io/badge/license-BSD--3--Clause-lightgrey.svg?style=flat-square)](https://opensource.org/licenses/BSD-3-Clause)  [![Gitter](https://img.shields.io/gitter/room/freeCodeCamp/Contributors.svg?style=flat-square)](https://gitter.im/freeCodeCamp/Contributors)
+
+[![GitHub Issues](https://img.shields.io/github/issues/freeCodeCamp/guides.svg?style=flat-square)](https://github.com/freeCodeCamp/guides/issues) [![GitHub Pull Requests](https://img.shields.io/github/issues-pr/freeCodeCamp/guides.svg?style=flat-square)](https://github.com/freeCodeCamp/guides/pulls) [![Pull Requests Bienvenida](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+[![Novatos Amigable](https://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](http://www.firsttimersonly.com/)
+
+[![Throughput Graph](https://graphs.waffle.io/freeCodeCamp/guides/throughput.svg)](https://waffle.io/freeCodeCamp/guides/metrics)
+
+La comunidad freeCodeCamp esta creando una **"Guia"** masiva.
+
+Esta herramienta de referencia eventualmente incluira miles de articulos que cubriran todas las areas de desarrollo, diseño y ciencia de datos - todo escrito para ser facilmente comprendido por personas nuevas en la programación.
+
+Este repositorio es donde planeamos y mantenemos estos articulos guias, que luego son dispuestan en la guia estilo wiki de la comunidad,[`guide.freecodecamp.org`](https://guide.freecodecamp.org).
+
+## ¿Qué son articulos guia?
+
+
+Los articulos guia buscan ser cortos articulos para iniciarte y familiarizarte con estos conceptos antes de leer en profundidad material mas especifico.
+
+Los articulos guias pueden ser explicaciones de sintaxis, patrones de diseño, para que sirven las etiquetas aria, o el significado de los numeros en la esquina superior derecha de tu pantalla en [freecodecamp.org](https://freecodecamp.org).
+
+Aca puedes encontrar un [articulo de ejemplo sobre elementos HTML ](./src/pages/html/elements/index.md).
+
+## ¿De qué temas puedo escribir un articulo?
+
+Tu ayuda escribiendo estos articulos es bienvenida.No hace falta que seas un experto in un topico para hablar de el - toda esta guia es open source, asi que incluso si cometes un error, otro contribuyente puede corregirla.
+
+Como ayuda, encuentra un `articulo reducido`
+To help, find a `stub article` on our [Guide website](https://guide.freecodecamp.org/), write the article, then open a pull request (PR) to replace the stub with your article.
+
+Si no encuentras un resumen del topico del que te gustaria escribir, puedes abrir un PR(Pull request) que cree el resumen e incluya tu borrador del articulo.
```

Comment:
Topico in this context doesn´t make sense we should use tema instead 
See [topico no es tema (in Spanish)](http://www.fundeu.es/recomendacion/topico-no-es-tema-materia-ni-asunto-1406/)

Reply:
Perfect augur, my spanish is not from Spain and we have some different meanings some times. I will correct it to fit the mistake.